### PR TITLE
nixpkg-review: rename and add missing flags

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: _nixpkgs-review [hunk 1]
```diff
@@ -20,9 +21,16 @@ local -a pr_args=(
     ':pr number to review:'
     '(-c --checkout)'{-c,--checkout}'[merge before building or use exact commit]:merge or commit:(merge commit)'
     "--eval[use ofborg's eval result]::(ofborg local)"
+    "--post-result[Post the nixpkgs-review results as a PR comment]"
     '--token[github token]:github token:'
 )
 
+local -a wip_args=(
+    '(-b --branch)'{-b,--branch}'[Branch to compare against with]'
```
Comment (hedning) [inline _nixpkgs-review:29]:
```suggestion
    '(-b --branch)'{-b,--branch}'[Branch to compare against with]:branch:__git_ref_specs'
```

`-b` takes an argument so should have `:argument_description:_completion_function`.

File: _nixpkgs-review [hunk 2]
```diff
@@ -20,9 +21,16 @@ local -a pr_args=(
...
+    '(-s --package)'{-s,--staged}'[Whether to build staged changes]'
```
Comment (hedning) [inline _nixpkgs-review:30]: Might've been a typo, but if `--package` and `--staged` can't be used at the same time then I believe you want something like this. Stuff in the parenthesis are patterns which should no longer complete when the option after is encountered.  
```suggestion
    '(-s --staged -p --package)'{-s,--staged}'[Whether to build staged changes]'
```

File: _nixpkgs-review [hunk 3]
```diff
@@ -20,9 +21,16 @@ local -a pr_args=(
...
+    '(-r --remote)'{-r,--remote}'[Name of the nixpkgs repo to review]'
```
Comment (hedning) [inline _nixpkgs-review:31]: Hmm, right, this one should also take an argument, not sure if it's possible to provide good completions though.

Comment (Mic92) [reply _nixpkgs-review]: Don't know.

**Comment by Mic92** [comment]
Fixed.

**Comment by hedning** [comment]
Cheers :)

**Pull request merged**